clusters/kind/cleanup.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-kind delete cluster
+kind delete cluster --name fats
```

Comment:
There is an existing envvar for the cluster name that is generated for each run.

```suggestion
kind delete cluster --name ${CLUSTER_NAME}
```